### DISCUSSION TITLE
Curiosity to your GTA raw data tool.

### DISCUSSION THREAD

**Discussion by jx2k6** [SysCV/qd-3dt#23] [Q&A]
Congrats to your awesome project!

Its says that 'we utilize in-game API to capture global instance id and corresponding 3D annotations directly' in your paper 'Monocular Quasi-Dense 3D Object Tracking'.
I wonder if the in-game API is a game mod or a script you wrote? Could I get access to the tool to generate raw data?

Thanks!

**Comment by eborboihuc** [comment]
Hi @jx2k6, thanks for your interest in our work!

Unfortunately, the tool is not publicly available. It's an internal tool developed in Berkeley Deep Drive. I believe it's built by Prof. 
Philipp Krähenbühl. This [repository](https://github.com/philkr/gamehook_gtav) might be helpful for your goal.

**Reply by jx2k6** [reply]
Hi @eborboihuc, thanks for your rapid reply!

I mistook https://download.visinf.tu-darmstadt.de/data/from_games/ as your tool and the code source is https://bitbucket.org/visinf/projects-2016-playing-for-data-renderdoc/src/playing_for_data/

Hope that I could figure a way through this!
